### ISSUE TITLE
On linux x64 bit (Fedora 10 x64) make install generates the si in /usr/lib/purple-2 instead of /usr/lib64/purple-2

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#115]
```
What steps will reproduce the problem?
1. Download tar.gz sources
2. make
3. # make install

What is the expected output? What do you see instead?
The resulting library (libmsn-pecan.so) to be generated in
/usr/lib64/purple-2 instead of /usr/lib/purple-2 which contains the 32 bit
libraries, so it will not be loaded by pidgin x64

What version of the product are you using? On what operating system?
Fedora 10 Linux 2.6.27.9-159.fc10.x86_64 #1 SMP Tue Dec 16 14:47:52 EST
2008 x86_64 x86_64 x86_64 GNU/Linux

Please provide any additional information below.


```

Original issue reported on code.google.com by `adrian.p...@gmail.com` on 12 Jan 2009 at 6:15

**Comment by GoogleCodeExporter** [comment]
```
I can see how that could happen. What is the output of this command?

pkg-config --variable=libdir purple
```

Original comment by `felipe.contreras` on 12 Jan 2009 at 6:19
- Changed state: **Accepted**
- Added labels: **Type-Defect**, **Priority-High**

**Comment by GoogleCodeExporter** [comment]
```
I'm not the original submitter, but...

On Fedora 9 at least, the result is "/usr/lib64". Haven't gotten a chance to 
upgrade
to F10 yet but I would expect the same result there.

```

Original comment by `joshua.t...@gmail.com` on 2 Feb 2009 at 8:35

**Comment by GoogleCodeExporter** [comment]
```
Thanks, then with that information it looks like using that variable would fix 
it. I
committed the fix and it will be on the next release (0.0.18).
```

Original comment by `felipe.contreras` on 5 Feb 2009 at 8:52
- Changed state: **Fixed**
